src/components/Tabs/useTabStyles.tsx

```diff
@@ -24,7 +24,7 @@ const useTabStyles = ({ index }: UseTabStylesProps): Partial<AbstractButtonProps
   }
 
   return {
-    outline: 'none',
+    zIndex: 1,
```

Comment:
I don't think that manipulating the items would do the trick.

Reply:
the `zIndex` is for an unrelated issue (where the border of the `Tab` should be above the pseudoelement of the `TabList`)